ci/pre-prod/build_image_role/tasks/main.yml

```diff
@@ -18,6 +18,23 @@
     dest: /home/firecrest/awx-firecrest-build
     version: "{{ commit_id }}"
 
+- name: build base image with latest tag 
```

Comment:
To play safe in the possible case of future concurrent builds (where I assume `latest` is much more likely to be shared than `build_tag`), I would do the other way around. Build `build_tag` first, and tag it as `latest` later.

Reply:
Fixed in commit `f6e9b60`